### PR TITLE
Game feature

### PR DESCRIPTION
**Pull request by fatymahmed**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Tic tac toe</title>
+    <link rel="stylesheet" href="/css/style.css">
```
Comment (AdnanAfsari) [inline index.html]: path is not correct. remove extra '/' from this line.

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
Hello @fatymahmed @Urchmaney ✋ 

**Changes Required**  

Nice work guys 👏but some small changes to make.

- [x] Check my comment on index.html regarding css file path and fix that.

- [x] Its better to clear up the winning message after reseting the game.It does not get clear now.have a look and fix that.
Screenshot 2019-10-22 at 1 39 28 AM

Please submit for another review after making those changes.

Happy Coding
Adnan Afsari

File: index.html (hunk 1 above)
Comment (fatymahmed) [reply index.html]: Thank you for the review, we've made the necessary changes

**Review by BalintMendli — APPROVED** [review]
Hi @fatymahmed @Urchmaney ,

Great work addressing issues from the previous review. 👏 

It is a
### 👍 

You can go ahead and merge the project.

**Pull request merged**